presto-docs/src/main/sphinx/release/release-0.184.rst

```diff
@@ -0,0 +1,41 @@
+=============
+Release 0.184
+=============
+
+General Changes
+---------------
+
+* Fix query execution failure for ``split_to_map(...)[...]``.
+* Fix issue that caused queries containing ``CROSS JOIN`` to not release CPU resources
+  even after they are killed.
+* Fix planning failure due to incorrect static evaluation of aggregation queries
+  with ``count(*)`` and non empty ``GROUP BY`` clause.
+* Fix communication failures caused by lock contention in new scheduler.
+* Add :func:`hamming_distance` function.
+* Improve performance of :func:`element_at` to be constant time rather than O(n).
```

Comment:
Is this for arrays, maps, or both? I'd be surprised if arrays were not already O(1).

Reply:
We should write this as "rather than proportional to the size of the map" (just as we write "constant time" rather than "O(1)"). 

Also, let's update the 0.180 release note line to use the same language.